management/management_api/endpoints/endpoint_utils.py

```diff
@@ -198,21 +195,19 @@ def list_endpoints(namespace: str, id_token: str):
 
 def get_endpoints_name_status(deployments, namespace):
     deployments = deployments.items
-    name_status = "There are no endpoints present in {} tenant".format(namespace)
+    endpoints_name_status = list()

```

Comment:
Not sure about this `name_status`. Took me a while to get that. Perhaps we could remove this or call it metadata?
`endpoint` or `endpoint_metadata`.